MicrosoftTeams.schema.json

```diff
@@ -1120,6 +1120,11 @@
           "type": "boolean",
           "description": "A boolean value indicating whether this app can stream the meeting's audio video content to an RTMP endpoint.",
           "default": false
+        },
+        "isAnonymousAccessAllowed": {
+          "type": "boolean",
+          "description": "A boolean value indicating whether this app allows management by anonymous users.",
```

Comment:
"management" or "access"? I think the description may be a bit misleading.

Reply:
Lets just align the description with that of the new property that will supersede it anyways.

"A boolean value indicating whether this app supports access by anonymous guest users."